src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs

```diff
@@ -19,11 +19,12 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("Storage", "BlobContainers", "2018-03-01-preview"),
-                new Tuple<string, string, string>("Storage", "Operations", "2018-03-01-preview"),
-                new Tuple<string, string, string>("Storage", "Skus", "2018-03-01-preview"),
-                new Tuple<string, string, string>("Storage", "StorageAccounts", "2018-03-01-preview"),
-                new Tuple<string, string, string>("Storage", "Usages", "2018-03-01-preview"),
+                new Tuple<string, string, string>("Storage", "BlobContainers", "2018-07-01"),
+                new Tuple<string, string, string>("Storage", "ManagementPolicies", "2018-03-01-preview"),
```

Comment:
Please ensure that one of the services is still preview

Reply:
Yes, ManagementPolicies feature is not supported in 2017-07-01 api version. So to keep the feature in SRP SDK, we add it from 2018-03-01-preview api version.